packages/subapp-react/test/spec/ssr-framework.spec.js

```diff
@@ -1,15 +1,36 @@
 "use strict";
-
 const url = require("url");
 const React = require("react"); // eslint-disable-line
 const lib = require("../../lib");
 const { withRouter } = require("react-router");
 const { Route, Switch } = require("react-router-dom"); // eslint-disable-line
-const { asyncVerify } = require("run-verify");
 const Redux = require("redux");
 const { connect } = require("react-redux");
+const { Stream } = require("stream");
 
 describe("SSR React framework", function () {
+  function getTestWritable() {
```

Comment:
style: be consistent on function style. This repo generally uses arrow functions.